css/progressbar.css

```diff
@@ -12,20 +12,20 @@
     position: relative;
     margin: 4px 10px 4px 35px;
     max-width: 1200px;
+    border: 2px solid #FFE238;
+    border-radius: 8px;
 }
 
 #donationMeter {
     margin: 0 auto;
     height: 20px;
     background: #FF6C00;
```

Comment:
Shouldnt the background also go to the wrapper? Otherwise it draws over the rounded borders...